sway/commands/output/dpms.c

```diff
@@ -1,45 +1,5 @@
 #include "sway/commands.h"
-#include "sway/config.h"
-#include "sway/output.h"
-#include "util.h"
-#include <strings.h>
 
 struct cmd_results *output_cmd_dpms(int argc, char **argv) {
```

Comment:
Nit: should print an error message to the user saying it's a deprecated function.